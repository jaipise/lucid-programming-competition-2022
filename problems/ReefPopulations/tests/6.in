100 50
196 27 134 24 48 192 193 62 99 177 128 147 32 187 27 112 5 98 42 137 57 32 47 57 130 174 69 86 77 88 37 151 24 21 77 40 21 24 121 110 192 75 33 151 45 172 12 44 16 147 31 197 74 183 103 131 92 58 89 15 161 141 102 166 5 86 184 99 35 102 83 82 171 181 42 186 116 109 183 19 65 42 91 53 115 61 52 14 179 45 119 52 71 108 111 127 167 43 187 99
9 96
28 63
73 100
28 87
61 86
32 68
98 100
25 66
9 31
19 43
33 80
51 64
25 29
81 81
28 72
64 68
44 51
77 82
46 48
74 98
74 87
25 82
83 99
14 28
59 82
95 100
15 98
19 36
50 72
85 94
41 86
53 95
36 62
56 72
14 32
94 99
99 100
81 84
90 95
23 86
58 86
46 95
6 84
10 97
63 71
80 83
54 79
39 81
8 92
34 73