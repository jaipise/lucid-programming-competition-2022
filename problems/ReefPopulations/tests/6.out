120
37
240
3
88
109
243
185
245
236
141
31
114
65
79
46
170
214
140
131
231
126
203
36
46
68
174
92
64
130
180
210
242
147
131
75
216
5
42
72
52
7
213
188
94
35
99
169
125
13
